### ISSUE TITLE
[AXE-CORE] HLR: Fix review page a11y issues

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#3878]
## Description

On the HLR review page, the axe browser extension reports some issues within the expanded accordions

* A `<dl>` elements is being returned by the `'ui:reviewField'`; this should be a `<div>`. Documentation has been updated in [this PR](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/192). This is an issue within the forwarding address & scheduled times review blocks.
* Contested issue review block:
  * The main block has a reported a11y issue with nested `<div>`s inside of a `<dl>`.
  * The followup block has a reported a11y issue of empty headers.
 
## Point of Contact

R. Garrison

## Acceptance Criteria

- [ ] No nested `<div>`s within a `<dl>`
- [ ] No empty headers
